### ISSUE TITLE
Experiment: Rollout

### ISSUE THREAD

**Issue by mateuszmatu** [mateuszmatu/OceanAI#252]
Once we have a model with a good loss function, we should do rollout training. 
The models generally look ok with inference for 2-3 time steps, but after that artifacts show up which "explode" over time. My hunch is that rollout will remove these artifacts to some extent.

**Comment by mateuszmatu** [comment]
Running rollout with 3 steps for 50 000 interations on 
learning-rate-v2/lr3-5e-04-st200-se42
The loss here was the best out of the learning rates we tried.

**Comment by mateuszmatu** [comment]
rollout seemed to improve the model performance quite well. Less artifacts. We should be doing rollout, but can play around with the setup for it.
